### PR TITLE
Update back-compat log with 11547 and 11536

### PR DESCRIPTION
**Pull request by jcouv**
Doc-only change to record two fixes that had some back-compat ramifications.
@dotnet/roslyn-compiler for review.

### CODE REVIEW

**Comment by jcouv** [comment]
@dotnet/roslyn-compiler for a doc-only review. Thanks

File: docs/Breaking API Changes.md [hunk 1]
```diff
@@ -15,3 +15,34 @@ Changed Simplifier.ReduceAsync, Simplifier.ExpandAsync, and Simplifier.Expand me
 
 PR: https://github.com/dotnet/roslyn/pull/5144
 
+# Version 1.3.0
+
+### Treat a method marked with both public and private flags as private
+
+The scenario is loading an assembly where some methods, fields or nested types had their accessibility flags set to 7 (all three bits set), which mean public AND private.
```
Comment (cston) [inline docs/Breaking API Changes.md]: Perhaps `have accessibility` rather than `had their accessibility`

**Comment by cston** [comment]
LGTM

**Comment by jcouv** [comment]
Thanks Chuck. I'll go ahead with your suggestion and merge.

**Comment by tannergooding** [comment]
test windows_eta_open_prtest please

**Pull request merged**